inc/sub-headerException.php

```diff
@@ -0,0 +1,124 @@
+<div class="newsSubHeader clearfix">
```

Comment:
Can you use dashes here instead of camelcase for css (unless it is existing code that would require additional work.) 
Exmaple: "news-subheader" instead of "newsSubheader" 
And same for below: innerPadding, socialNav, singleSocialNav